package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dvf-client-js",
-  "version": "3.1.8",
+  "version": "3.2.0",
```

Comment:
Should this be 4.0.0 as it’s a breaking change?

Reply:
Makes sense to be clear we are on starkex v4

Reply:
I did originally set this to 4.0.0, but then reverted it as functions were already broken. But a fair point this a clear break in the interface, will bump again to 4.0.0